src/modules/75-cf/strings/strings.en.yaml

```
@@ -492,6 +492,8 @@ gitSync:
   tryAgain: Try Again
   goToGit: Go to Git to fix issue
   invalidYaml: 'We are unable to render your {{{ flagsYamlFilename }}} file because it is invalid. Fix the following issues on Git and try again.'
+policyEvaluations:
+  failedToSave: 'Flag cannot be saved due to the policy evaluation failures'
```

Not a problem, but you don't need quotes in YAML unless it contains breaking characters
```suggestion
  failedToSave: Flag cannot be saved due to the policy evaluation failures
```

Works either way, so please feel free to ignore